transcode/transcode.go

```diff
@@ -385,6 +406,17 @@ func getHlsTargetURL(tsr TranscodeSegmentRequest) (*url.URL, error) {
 	}
 }
 
+func getFmp4TargetURL(tsr TranscodeSegmentRequest) (*url.URL, error) {
+	if tsr.FragMp4TargetUrl != "" {
```

Comment:
nit: reverse the logic and do `return nil, nil` here

Reply:
Will change it.